CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 # ref: https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
 
-* @shashankram @snehachhabria @nojnhuh @draychev @eduser25
+* @shashankram @snehachhabria @nojnhuh @draychev
 
 # Emeritus maintainers
-# @michelleN
\ No newline at end of file
+# @michelleN
```

Comment:
You should be added as an emeritus maintainer!

```suggestion
# @michelleN @eduser25
```

Reply:
thanks Thomas